pic32/cores/pic32/WInterrupts.c

```diff
@@ -279,11 +279,7 @@ void detachInterrupt(uint8_t interruptNum)
 #ifdef PIN_INT0
 //************************************************************************
 // INT0 ISR
-#if defined(__PIC32MZXX__)
-void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS))) ExtInt0Handler(void)
```

Comment:
Matt, the __USER_ISR() macro does not capture the at_vector() or interrupt() parts of this __attribute__. Is it really OK to replace these with __USER_ISR?